### ISSUE TITLE
EventEmitter memory leak

### ISSUE THREAD

**Issue by didodido85** [fluent/fluent-logger-node#43]
Following this:

https://github.com/nodejs/node-v0.x-archive/issues/5108

i can't set maxListener to FluentSender.

**Comment by okkez** [comment]
You can do that by following code.

```js
var logger = require('fluent-logger').createFluentSender('tag', {
  host: 'localhost',
  port: 24224,
  timeout: 3.0
});

logger._eventEmitter.setMaxListeners(10);
console.log(logger._eventEmitter);
```

Do you need public API for this?

**Comment by didodido85** [comment]
Trying to do that.

Receive:

logger._eventEmitter.setMaxListeners(config.fluentMaxListner);
                                  ^
TypeError: Cannot call method 'setMaxListeners' of undefined

**Comment by okkez** [comment]
I tested following versions of node:

* v0.10.41
* v0.12.9
* v4.2.1
* v5.1.1

Could you tell me your environment?

**Comment by okkez** [comment]
Released v0.5.0

**Comment by didodido85** [comment]
Thx
